docs/scenarios/v2x.rst

```diff
@@ -18,18 +18,21 @@ Build V2X module
 
 .. code-block:: bash
 
-   cd zenoh_autoware_v2x && poetry install --no-root
+   cd autoware_carla_launch
+   source env.sh
+   cd external/zenoh_autoware_v2x
+   poetry env use $(pyenv which python) && poetry install --no-root
```

Comment:
You can move the installation to here
https://github.com/evshary/autoware_carla_launch/blob/main/script/dependency_install.sh#L17